packages/compass-workspaces/src/components/index.tsx

```diff
@@ -115,6 +116,11 @@ const WorkspacesWithSidebar: React.FunctionComponent<
   useEffect(() => {
     onChange.current(activeTab, activeTabCollectionInfo);
   }, [activeTab, activeTabCollectionInfo, onChange]);
+  useSyncAssistantGlobalState('currentWorkspace', activeTab);
+  useSyncAssistantGlobalState(
+    'currentWorkspaceCollectionInfo',
+    activeTabCollectionInfo
+  );
```

Comment:
I think compass-collection plugin is a better place for setting up this subscription: this state is still here for now, but chances are high we will drop it as it's not used by anything and strictly speaking doesn't need to be stored in workspaces plugin (it was added here initially because we used it to render tab title, this is now done by the workspaces themselves)